descriptors/ModuleDescriptor-template.json

```diff
@@ -6,6 +6,16 @@
       "id": "circulation-bff-requests",
       "version": "1.0",
       "handlers": [
+        {
+          "methods": [
+            "GET"
+          ],
+          "pathPattern": "/circulation-bff/external-users/{externalUserId}/tenant/{tenantId}",
+          "permissionsRequired": ["circulation-bff.externalUser.get"],
```

Comment:
Is this permission name Eureka-compatible?
BTW, it definitely shouldn't contain uppercase letters.

Reply:
fixed